config/main.py

```diff
@@ -977,6 +977,12 @@ def _stop_services():
     except subprocess.CalledProcessError as err:
         pass
 
+    # Get the list of dependencies for sonic.target to fetch timer units
+    for service in _get_sonic_services():
+        if service.endswith('.timer'):
```

Comment:
>service [](http://example.com/codeflow?start=11&length=7)

Just curios, these services are already dependencies of sonic.target. Why original code does not stop the them?